src/queue.ts

```diff
@@ -31,7 +32,7 @@ export class Queue<TItem> extends EventEmitter {
     }).promise()
   }
 
-  startProcessing (handler : (item : TItem) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
+  startProcessing (handler : (item : TItem, message : any) => any | PromiseLike<any>, options?: ProcessOptions): PromiseLike<void> {
```

Comment:
I don't like the `message` variable with type `any`. Can you pass in the correct type of the message?

Maybe it will be necessary to create the interface by hand, with all of the `message` objects methods and properties.